### ISSUE TITLE
Errors during installation

### ISSUE THREAD

**Issue by Smuger12** [ryanrudolfoba/SteamOS-Waydroid-Installer#45]
### I get these errors during installation:
```
SteamOS Waydroid Installer Script by ryanrudolf
https://github.com/ryanrudolfoba/SteamOS-Waydroid-Installer
Script is running in Desktop Mode.
Checking if kernel is supported.
6.1.52-valve9-1-neptune-61 is supported. Proceed to next step.
Please enter current sudo password:
Checking if the sudo password is correct.
sudo password is good!
Cloning to "/home/deck/AUR/waydroid/waydroid_script"...
remote: Enumerating objects: 717, done.
remote: Counting objects: 100% (327/327), done.
remote: Compressing objects: 100% (128/128), done.
remote: Total 717 (delta 259), reused 238 (delta 199), pack-reused 390
Object download: 100% (717/717), 21.74 MiB | 788.00 KiB/s, done.
Delta resolution: 100% (437/437), done.
Warning: The rootfs is already read-write!
          Nothing is performed.
==> Adding keys from archlinux.gpg...
==> Adding keys from holo.gpg...
==> Updating trusted database...
gpg: the next database check will take place on 2024-04-10
pacman keyring has been initialized!
cp: cannot create regular file '/lib/modules/6.1.52-valve9-1-neptune-61/binder_linux.ko.zst': Read-only file system
Error installing binder kernel module. Goodbye!
rm: cannot remove '/lib/modules/6.1.52-valve9-1-neptune-61/binder_linux.ko.zst': No such file or directory
```

**Comment by ryanrudolfoba** [comment]
This looks like you have other scripts / tools interfering with the installer script. Script is created and tested on stock SteamOS Stable and stock SteamOS Preview.

Check your system for any tools / scripts that prevents disable read-only.


cp: cannot create regular file '/lib/modules/6.1.52-valve9-1-neptune-61/binder_linux.ko.zst': Read-only file system

**Comment by Smuger12** [comment]
I have these tools installed: CryoUtilities (restored to stock) and Decky, so what can prevent the script from turning off readonly (I can turn off readonly myself, but not the script?).

**Comment by ryanrudolfoba** [comment]
The script turns off the readonly but for some reason on your system it cant copy files and still complains about the readonly file system. I've tested the script multiple times on my OLED and LCD, and several users have reported success too.

To troubleshoot your system try this -

turn off the readonly and manually copy the binder kernel module to -

`/lib/modules/6.1.52-valve9-1-neptune-61/binder_linux.ko.zst `

**Comment by Smuger12** [comment]
Even if I turn off readonly and manually copy the binder_linux.ko.zst file, the problem is still the same.

**Comment by ryanrudolfoba** [comment]
> Even if I turn off readonly and manually copy the binder_linux.ko.zst file, the problem is still the same.

Yes so its not a problem with the script. Something installed on your system is preventing the files to be copied.

**Comment by NickoNick-dot** [comment]
so then what do we do?

**Comment by NickoNick-dot** [comment]
cuz its getting annoying

**Comment by ryanrudolfoba** [comment]
> so then what do we do?

Yup its not a problem on the script itself its an issue with your SteamOS install. It can really be annoying as i dunno what you have installed etc etc

**Comment by ryanrudolfoba** [comment]
> I have EmuDeck and Decky installed

Unfortunately I'm not able to fix your SteamOS issue. Something is wrong with your SteamOS install preventing the script to run successfully.

I'm not able to replicate your particular SteamOS issue either ever since I started to write this script more than 1yr ago.

**Comment by ryanrudolfoba** [comment]
> I have EmuDeck and Decky installed

i've received some feedback that this particular error is caused by Decky Loader.

i'll update the script to temporarily disable Decky Loader during installation so it doesnt interfere with the waydroid installation.

**Comment by ryanrudolfoba** [comment]
script has been updated to add Decky Loader sanity check -
https://github.com/ryanrudolfoba/SteamOS-Waydroid-Installer/commit/5b1ef477dd72158544e26a06ef5609cb625ea8fc

if Decky Loader is detected it will temporary disable the Plugin Loader service and proceed with the install.

Clone the repo to get the latest version of the script.
